app/views/tickets/_ticket.html.haml

```diff
@@ -9,7 +9,7 @@
           = markdown(ticket.description)
   %td.col-sm-1.col-md-1
     - if ticket.bought?(current_user)
-      = text_field_tag("tickets[][#{ticket.id}]", ticket.quantity_bought_by(current_user),
+      = text_field_tag("tickets[][#{ticket.id}]", 0,
```

Comment:
this `text_field_tag` looks exactly like the one on line 15.. why do we need the `if`?